source/scripts/createPassword.js

```diff
@@ -0,0 +1,32 @@
+function init() {
+    const button = document.querySelector('button');
+    const input = document.getElementById('password');
+    const alert = document.getElementById('strongPassword');
+    const passwordPage = document.getElementById('passwordPage');
+    const hint = document.getElementById('hint');
+    let password;
+    let pin;
+    button.addEventListener('click', () => {
+        if (input.value.length < 6) {
```

Comment:
Does this mean that the pin can only be less than 6 characters long? Do you guys think we can have a variable that keeps track of whether the user is entering the pin or the password and then check that variable in this if statement instead?